src/options.js

```diff
@@ -31,6 +32,7 @@ export type Options = {|
   residual?: boolean,
   serialize?: boolean,
   strictlyMonotonicDateNow?: boolean,
+  errorHandler?: ErrorHandler,
```

Comment:
I've changed this to be passed independently of Options in prepack-standalone, which is needed for tests.
To make it useful for prepack-cli or any other consumer, this will need to be added to prepack-node.js as well. It's easy to do, but since it's not very useful at the moment, I figured it's better to leave it out for now to not mess up the interface with unnecessary parameters for now. Let me know if you think otherwise and want me to introduce it there as well.